cookbooks/aws-parallelcluster-config/spec/unit/recipes/sudo_config_spec.rb

```diff
@@ -28,7 +28,7 @@
           runner = ChefSpec::Runner.new(platform: platform, version: version) do |node|
             node.override['cluster']['node_type'] = 'HeadNode'
             node.override['cluster']['dcv_enabled'] = 'head_node'
-            node.override['conditions']['dcv_supported'] = true
+            allow(File).to receive(:exist?).with('/etc/dcv/dcv.conf').and_return(true)
```

Comment:
You should mock the function, not the file existence.

Reply:
done. I missed to update it when I added the function definition in the latest commit.